config/default.json

```diff
@@ -13,5 +13,15 @@
     "host": "localhost",
     "port": 5672,
     "exchange": "harvests"
+  },
+  "keycloak": {
+    "realm": "fdk",
+    "bearer-only": true,
+    "auth-server-url": "http://localhost:8084/auth",
+    "ssl-required": "external",
+    "resource": "fdk-harvest-admin",
+    "verify-token-audience": true,
+    "use-resource-role-mappings": true,
+    "confidential-port": 0
```

Comment:
Make sure only required properties are set. My guess is that out of all the properties, the ones you need are:
- realm
- resource
- verify-token-audience